messages/lightning.preview.app.md

```diff
@@ -71,74 +69,79 @@ Downloading
 # certificate.installation.steps.android.nav-target-api-28
 
-Navigate to `Settings > Security & Location > Advanced > Encryption & credentials` and tap on `Install from SD card` under `Credential storage`.
+Navigate to `Settings > Security & Location > Advanced > Encryption & credentials` and click `Install from SD card` under `Credential storage`.
 
 # certificate.installation.steps.android.nav-target-api-29
 
-Navigate to `Settings > Security > Encryption & credentials` and tap on `Install from SD card` under `Credential storage`.
+Navigate to `Settings > Security > Encryption & credentials` and click `Install from SD card` under `Credential storage`.
 
 # certificate.installation.steps.android.nav-target-api-30-32
 
-Navigate to `Settings > Security > Encryption & credentials` and tap on `Install a certificate` under `Credential storage`. Now tap on `CA certificate` and on `Install anyway`.
+Navigate to `Settings > Security > Encryption & credentials` and click `Install a certificate` under `Credential storage`. Click `CA certificate`, and then click `Install anyway`.
 
 # certificate.installation.steps.android.nav-target-api-33
 
-Navigate to `Settings > Security > More security settings > Encryption & credentials` and tap on `Install a certificate` under `Credential storage`. Now tap on `CA certificate` and on `Install anyway`.
+Navigate to `Settings > Security > More security settings > Encryption & credentials` and click `Install a certificate` under `Credential storage`. Click `CA certificate`, and then click `Install anyway`.
 
 # certificate.installation.steps.android.nav-target-api-34-up
 
-Navigate to `Settings > Security & Privacy > More security & privacy > Encryption & credentials` and tap on `Install a certificate` under `Credential storage`. Now tap on `CA certificate` and on `Install anyway`.
+Navigate to `Settings > Security & Privacy > More security & privacy > Encryption & credentials` and click `Install a certificate` under `Credential storage`. Click `CA certificate`, and then click `Install anyway`.
 
 # certificate.waiting
 
-After installing the certificate, press any key to continue...
+After you install certificate, press any key to continue...
 
 # mobileapp.notfound
 
-%s is not installed on your device.
+%s isn't installed on your device.
 
 # mobileapp.download
 
-%s is not installed on your device. Do you want to download and install it
+%s isn't installed on your device. Do you want to download and install it?
 
 # examples
 
-- <%= config.bin %> <%= command.id %>
+- Preview the target org "myOrg" in a desktop environment.
+  <%= config.bin %> <%= command.id %> --target-org myOrg
+- Preview the application "myApp" for the target org "myOrg" in a desktop environment:
+  <%= config.bin %> <%= command.id %> --name MyApp --target-org myOrg --device-type desktop
+- Preview the target org "myOrg" on an iOS device:
+  <%= config.bin %> <%= command.id %> --target-org myOrg --device-type ios --device-id "37C98F87-63C4"
```

Comment:
@maliroteh-sf Should our `--device-id` value in this case be one of the friendly names of the devices, which we also allow for? E.g. `--device-id "iPhone 15 Pro Max"`? Users are simply going to run away screaming from the example given. 😬

Reply:
Oh yes that's a good point. We should switch to using the friendly name in the examples instead, since our code works with both friendly name and ID.

Reply:
```suggestion
  <%= config.bin %> <%= command.id %> --target-org myOrg --device-type ios --device-id "iPhone 15 Pro Max"
```